govcd/edgegateway.go

```diff
@@ -737,3 +737,18 @@ func (eGW *EdgeGateway) AddIpsecVPN(ipsecVPNConfig *types.EdgeGatewayServiceConf
 	return *task, nil
 
 }
+
+// Removes an Edge Gateway VPN, by passing an empty configuration
```

Comment:
Link to documentation would be good if available.

Reply:
The documentation is thin about this operation. I'm open to suggestions.

Reply:
Nope.